worker/celery_tasks.py

```diff
@@ -87,6 +88,12 @@ def log_error(request, exc, traceback, credit_transfer_id):
         blockchain_processor.send_blockchain_result_to_app(result)
 
 
+@celery_app.task()
+def send_directory_listing(recipient_phone, user_input):
```

Comment:
@enjeyw is this one giant file that has all the celery tasks ever? why not put these directly in blockchain_tasker and misc_tasker?